tests/policies/water/LowWaterPolicy.cpp

```diff
@@ -1,6 +1,14 @@
+#define DOCTEST_CONFIG_IMPLEMENT_WITH_MAIN
 #include "doctest.h"
+#include "LowWaterPolicy.h"
```

Comment:
Also fix the import here it should be #include "policies/water/LowWaterPolicy.h" and not #include "LowWaterPolicy.h"